src/macos.rs

```diff
@@ -87,6 +87,8 @@ impl GlContext {
             let view = NSOpenGLView::alloc(nil)
                 .initWithFrame_pixelFormat_(parent_view.frame(), pixel_format);
 
+            view.setWantsBestResolutionOpenGLSurface_(YES);
```

Comment:
Could you move this to after the nil check?

Reply:
Fixed